TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -111,11 +113,13 @@ def makeTreeFromMiniAOD(
     from TreeMaker.WeightProducer.getWeightProducer_cff import getWeightProducer
     process.WeightProducer = getWeightProducer(process.source.fileNames[0])
     process.WeightProducer.Lumi                       = cms.double(1) #default: 1 pb-1 (unit value)
-    process.WeightProducer.PU                         = cms.int32(0) # PU S10 3 for S10 2 for S7
-    process.WeightProducer.FileNamePUDataDistribution = cms.string("NONE")
+    process.WeightProducer.PU                         = cms.int32(4) # PU S10 3 for S10 2 for S7
```

Comment:
Can you fix the comment on this line so it makes sense with our current procedure?